sandboxed_containers/deploying-sandboxed-container-workloads.adoc

```diff
@@ -1,16 +1,39 @@
 [id="deploying-sandboxed-containers-workloads"]
-= Deploying OpenShift sandboxed containers workloads
+= Deploying {sandboxed-containers-first} workloads
 include::modules/common-attributes.adoc[]
 :context: deploying-sandboxed-containers
 
 toc::[]
 
-include::modules/sandboxed-containers-installing-operator.adoc[leveloffset=+1]
-include::modules/sandboxed-containers-preparing-openshift-cluster.adoc[leveloffset=+2]
+You can install the {sandboxed-containers-operator} using either the web console or OpenShift CLI (`oc`). Before installing the {sandboxed-containers-operator}, you must prepare your {product-title} cluster.
+
+include::modules/sandboxed-containers-preparing-openshift-cluster.adoc[leveloffset=+1]
+
+.Additional Resources
+* xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[Installing a cluster on bare metal]
```

Comment:
This links to _Installing a user-provisioned cluster on bare metal_. The preparing section says: _Your cluster must use installer-provisioned infrastructure._ The next bullet here goes to the baremetal IPI instructions. Do we need this one?